native/src/modules/endpoint/reducers/cityContentReducer.js

```diff
@@ -17,14 +17,16 @@ export default (
   if (action.type === 'FETCH_CATEGORY') {
     const { language, path, depth, key, city } = action.params
     const initializedState = state || createCityContent(city)
-    const oldContent = state && state.categoriesRouteMapping[key] ? state.categoriesRouteMapping[key] : {}
+    const reuseOldContent = state && state.routeMapping[key] && state.routeMapping[key].routeType === 'category'
+    const oldContent = reuseOldContent ? state?.routeMapping[key] : {}
```

Comment:
```suggestion
    const reuseOldContent = state && state.routeMapping[key] && state.routeMapping[key].routeType === 'category'
    const oldContent = reuseOldContent ? state.routeMapping[key] : {}
```
will not work here as state is not-null only implicitly. Imo still fine. 
Same goes for `FETCH_EVENT` below

Reply:
Do you know why we don't need that kind of logic for `news` and `pois`?

Reply:
Okay, thanks for researching. I think that is fine for now.